pkg/api/handlers/compat/containers.go

```diff
@@ -495,6 +497,17 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 		exposedPorts[exposedPort] = struct{}{}
 	}
 
+	var healthcheck *container.HealthConfig
```

Comment:
maybe make this a separate function like `Hostname` or `Command` within `libpod/container.go`